pkg/controller/operators/olm/operator_test.go

```diff
@@ -3091,6 +3092,74 @@ func TestTransitionCSV(t *testing.T) {
 	}
 }
 
+func TestCaBundleRetrevial(t *testing.T) {
+	logrus.SetLevel(logrus.DebugLevel)
+	namespace := "ns"
+
+	type initial struct {
+		csvs []runtime.Object
+		crds []runtime.Object
+	}
+	type expected struct {
+		caBundle []byte
+		err      error
+	}
+	tests := []struct {
+		name     string
+		initial  initial
+		expected expected
+	}{
+		{
```

Comment:
Do we want to have a test case here for a case where more than just a conversion webhook is included? Or error cases?

Otherwise this lgtm

Reply:
Great idea - I added cases for validating and mutating webhooks and errors getting the CA.